platform/machine/hetzner/cluster.go

```diff
@@ -31,13 +34,14 @@ func (bc *cluster) NewMachine(userdata *conf.UserData) (platform.Machine, error)
 	// Hack to workaround CT inheritance.
 	// Can be dropped once we remove CT dependency.
 	// https://github.com/flatcar/Flatcar/issues/1386
+	// Does this need a workaround for https://github.com/coreos/afterburn/pull/1083?
+	// The COREOS_COREOS line is a workaround for https://github.com/coreos/afterburn/pull/1083
 	conf.AddSystemdUnitDropin("coreos-metadata.service", "00-custom-metadata.conf", `[Service]
 ExecStartPost=/usr/bin/sed -i "s/HETZNER/CUSTOM/" /run/metadata/flatcar
-ExecStartPost=/usr/bin/sed -i "s/IPV4_PRIVATE/PRIVATE_IPV4/" /run/metadata/flatcar
-ExecStartPost=/usr/bin/sed -i "s/IPV4_PUBLIC/PUBLIC_IPV4/" /run/metadata/flatcar
```

Comment:
I was not sure why these lines were included. Afterburn Hetzner already has `PUBLIC_IPV4` and `PRIVATE_IPV4` attributes.